synchronizer/templates/validation.html

```diff
@@ -18,18 +18,25 @@
                         <th>Stopped</th>
                         <th>Duration</th>
                         <th>Issue ID</th>
+                        <th>Correct ID</th>
                         <th>Actions</th>
                     </tr>
                 </thead>
                 <tbody>
                     {% for worklog in worklogs %}
-                    <tr class="{% if worklog.is_valid %} table-success {% else %} table-danger {% endif %}">
+                    <tr class="{% if worklog.correct_id %} table-success {% else %} table-danger {% endif %}">
                         <th scope="row">{{loop.index}}</th>
                         <td>{{worklog.comment}}</td>
                         <td>{{local_time(worklog.date_started)}}</td>
                         <td>{{local_time(worklog.date_stopped)}}</td>
                         <td>{{seconds_to_time(worklog.duration)}}</td>
                         <td>{{worklog.issue_id}}</td>
+                        <td>{{worklog.correct_id}}
+                            {% if not worklog.correct_id %}
+                                <span data-toggle="popover-hover" data-placement="top" title="Issue with this issue id is not present in Jira">

```

Comment:
What about other errors? In future we can add some new types of errors (like no access etc.).